cmd/publishing-bot/publisher.go

```diff
@@ -610,14 +608,14 @@ func (p *PublisherMunger) ensureCloned(dst string, dstURL string) error {
 
 // constructs all the repos, but does not push the changes to remotes.
 func (p *PublisherMunger) construct() error {
-	kubernetesRemote := filepath.Join(p.k8sIOPath, "kubernetes", ".git")
+	baseRemote := filepath.Join(p.baseRepoPath, p.config.SourceRepo, ".git")
```

Comment:
should we call this `source*` everywhere instead of `base*`, `upstream*`, `kubernetes*`? This leads to confusion.